.github/workflows/build.md

```diff
@@ -0,0 +1,10 @@
+## build.yml
+
+Runs holochain tests:
```

Comment:
i find it too early to document it in such detail, if at all. it requires manual updating of the markdown file every time tests are changed or relocated